api/tools/src/main/java/org/pmiops/workbench/tools/CreateTerraMethodSnapshot.java

```diff
@@ -208,6 +200,9 @@ public CommandLineRunner run() {
               + "/"
               + methodResponse.getSnapshotId()
               + "\n\n"
+              + "Using source repo commit "
```

Comment:
This is the change.  We should record this value in Jira or elsewhere to establish method provenance, for example: https://precisionmedicineinitiative.atlassian.net/browse/RW-14006?focusedCommentId=229621